documentation/meta/media_properties/frontend.md

```diff
@@ -0,0 +1,126 @@
+# Frontend Media Properties
+
+_This document is auto-generated from the source code in
+[/frontend/scripts/document-media.js](https://github.com/WordPress/openverse/blob/main/frontend/scripts/document-media.js)._
+
+This is a list of the media properties, with the descriptions of corresponding
+TypeScript types that are used in the frontend to hold media objects.
+
+## General notes
+
+The columns are sorted alphabetically. Both interfaces `ImageDetail` and
+`AudioDetail` inherit from interface `Media` so the tables for those interfaces
+only show their additional fields.
+
+Some types in the "Type" column may refer to interfaces or types defined in the
+frontend codebase. In such cases it is advised to refer to the source file
+[/frontend/src/types/media.ts](https://github.com/WordPress/openverse/blob/main/frontend/src/types/media.ts)
+and trace the source of the type definitions. Such types are italicised for
+clarity.
+
+## Interface `Media`
+
+| Name                  | Type                   | Optional? |
+| --------------------- | ---------------------- | --------- |
+| `attribution`         | `string`               |           |
```

Comment:
Same note here RE: linking to the field below!

Reply:
Implemented in c976546.